scripts/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM nvidia/cuda:11.3.1-devel-ubuntu20.04
+
+LABEL maintainer "DECI.AI <services@deci.ai>"
+ARG VERSION=master
```

Comment:
master is not a version. We never use master

Reply:
stable?

Reply:
tag, always tag